oak/proto/policy.proto

```diff
@@ -47,3 +48,11 @@ message GrpcTag {
   // https://github.com/google/asylo/blob/3158887cb768112516424b3e65046f3946eb4465/asylo/identity/identity.proto#L40.
   bytes authorization_bearer_token = 1;
 }
+
+// Policies related to modules, referring to the native WebAssembly node within the TCB.
+message ModuleTag {
```

Comment:
I think the name should make it clear that this is related to WebAssembly modules specifically. How about `WebAssemblyModuleTag`?

Reply:
Done.